### PR TITLE
Update README.md with execution tests, java tests and pipelines

### LINKED ISSUE #175: Update README
- Add Java compilation tests
- Add execution tests
- Describe pipelines 
[AB#8013](https://microsoftgraph.visualstudio.com/0985d294-5762-4bc2-a565-161ef349ca3e/_workitems/edit/8013)

### PR DESCRIPTION
**Pull request by fey101**
This PR updates README.md with execution tests, java tests and pipelines
Closes #175

### CODE REVIEW

**Review by baywet — APPROVED** [review]
I'd love to see the suggested launch.json somewhere in here too if we could since it's been removed from the repo by @zengin the other day.

**Comment by zengin** [comment]
@baywet that was for the generator repo and it was the `launchSettings.json` file which is for Visual Studio only as opposed to VSCode's `launch.json`. I will update that repo soon with my proposal.

File: README.md [hunk 1]
```diff
@@ -2,29 +2,54 @@
...
+2. CsharpBetaKnownFailureTests
+3. CsharpV1Tests
+4. CsharpV1KnownFailureTests
+
+5. CsharpBetaExecutionTests
+6. CsharpV1ExecutionTests
+
+7. CSharpArbitraryDllTests
+
+ The arbitraryDllTest is useful in running customized tests, for example where there is a need to use a modified unpublished dll for whichever repository to test the effect of a specific fix or assumed hypothesis.
```
Comment (zengin) [inline README.md:32]:
```suggestion
 The arbitraryDllTest is useful in running customized tests for an unpublished dll, which can consist of a proposed metadata or generator changes.
```

File: README.md [hunk 2]
```diff
@@ -34,3 +59,32 @@ Tests compile C# snippets, which based on HTTP snippets in Microsoft Graph docum
      - `Microsoft.Graph`
      - `Microsoft.Graph.Beta`
    - Adding references to individual queries are not necessary, as the `.linq` file that Raptor generates includes correct NuGet package referenced.
+
+
+## Pipelines
+The repository also contains a couple of CI pipelines. The CI pipelines run the tests outlined above and the output of these tests are then used to report success or failure rates in a graphical and analytical approach.
+The pipelines are available [here](https://microsoftgraph.visualstudio.com/Graph%20Developer%20Experiences/_build?view=folders&treeState=XFJhcHRvcg%3D%3D)
```
Comment (zengin) [inline README.md:66]: May be useful to note that this is in a private instance.
```suggestion
The pipelines are running in a private Azure DevOps instance [here](https://microsoftgraph.visualstudio.com/Graph%20Developer%20Experiences/_build?view=folders&treeState=XFJhcHRvcg%3D%3D)
```

File: README.md [hunk 3]
```diff
@@ -34,3 +59,32 @@ Tests compile C# snippets, which based on HTTP snippets in Microsoft Graph docum
...
+- Beta C# Snippets - Known Issues
+
+And the equivalent pipelines for running java tests are
+- V1 Java Snippet Compilation Tests
+- Beta Java Snippet Compilation Tests
+
+- Beta Java Snippet Compilation Tests - Known Issues
+- V1 Java Snippet Compilation Tests - Known Issues
+
+The scheduled pipelines are categorized into daily and weekly schedules. A single scheduled pipeline can contain a mix of categories of tests, for example the `PROD Beta Compilation - Daily`, runs "Beta C# Snippets" excluding known issues and after runs "Beta C# Snippets - Known Issues".
```
Comment (zengin) [inline README.md:90]:
```suggestion
The scheduled pipelines are categorized into daily and weekly schedules. A single scheduled pipeline can contain both categories of tests in a single run to report all failures including known issues. Azure DevOps tooling allows us to have these categories reflected in the test results page.
```

**Review by zengin — COMMENTED** [review]
Thanks for updating the README. I have a few minor suggestions.

**Comment by baywet** [comment]
@zengin sorry about that, yet another case of Vincent being confused between repos and files 🤪

**Review by baywet — APPROVED** [review]

**Review by baywet — APPROVED** [review]

**Pull request merged**